docs/index.md

```diff
@@ -0,0 +1,47 @@
+The Self Signed Certificates Operator provides self-signed X.509 certificates to your charms.
```

Comment:
Ultimately I think most of our docs for this simple charm can be in this index. It's a fairly simple charm at the end of the day and we probably don't want to maintain 5-10 pages of text for it.